e2e/utils.go

```diff
@@ -31,17 +31,18 @@ import (
 	testutils "k8s.io/kubernetes/test/utils"
 )
 
+/* #nosec:G101, credentials not vulnerable.*/
```

Comment:
Actually, these are not credentials, but pointers to the location of the credentials.